src/main/java/com/revature/studyforce/flashcard/dto/VoteDTO.java

```diff
@@ -0,0 +1,43 @@
+package com.revature.studyforce.flashcard.dto;
+
+import com.revature.studyforce.flashcard.model.Vote;
+import lombok.*;
+import org.springframework.util.Assert;
+import java.util.function.Function;
+
+/**
+ * Data Transfer Object to carry the Vote model's information to and from the front end
+ * {@link Vote}
+ * @author Elizabeth Ye
+ */
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+public class VoteDTO {
+
+    /**
+     * Fields used in UI layer
```

Comment:
properties should not be documented, instead use good naming convention to indicate their purpose